code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -884,3 +884,65 @@
 
 /datum/reagent/toxin/mimesbane/on_mob_delete(mob/living/L)
 	L.remove_trait(TRAIT_EMOTEMUTE, id)
+
+/datum/reagent/toxin/bonehurtingjuice //oof ouch
+	name = "Bone Hurting Juice"
+	id = "bonehurtingjuice"
+	description = "A strange substance that looks a lot like water. Drinking it is oddly tempting. Oof ouch."
+	color = "#AAAAAA77" //RGBA: 170, 170, 170, 77
+	toxpwr = 0
+	taste_description = "bone hurting"
+	overdose_threshold = 50
+	metabolization_rate = 2
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_add(mob/living/carbon/M)
+	M.say("oof ouch my bones", forced = "bonehurtingjuice")
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(15, 0)
+	if(M.dna.species.type == /datum/species/skeleton || M.dna.species.type == /datum/species/plasmaman)
+		M.adjustBruteLoss(0.5, 0)
+	if(prob(20))
+		switch(rand(1, 3))
+			if(1)
+				var/list/possible_says = list("oof", "ouch", "my bones", "oof ouch", "oof ouch my bones")
+				M.say(pick(possible_says), forced = "bonehurtingjuice")
+			if(2)
+				var/list/possible_mes = list("oofs softly", "looks like their bones hurt", "grimaces, as though their bones hurt")
+				M.say("*custom " + pick(possible_mes), forced = "bonehurtingjuice")
+			if(3)
+				to_chat(M, "<span class='warning'>Your bones hurt!</span>")
+	return ..()
+
+/datum/reagent/toxin/bonehurtingjuice/overdose_process(mob/living/carbon/M)
+	if(prob(4) && iscarbon(M)) //big oof
+		var/selected_part
+		switch(rand(1, 4)) //God help you if the same limb gets picked twice quickly.
+			if(1)
+				selected_part = BODY_ZONE_L_ARM
+			if(2)
+				selected_part = BODY_ZONE_R_ARM
+			if(3)
+				selected_part = BODY_ZONE_L_LEG
+			if(4)
+				selected_part = BODY_ZONE_R_LEG
+		var/obj/item/bodypart/bp = M.get_bodypart(selected_part)
+		if(M.dna.species.type == /datum/species/skeleton || M.dna.species.type == /datum/species/plasmaman) //We're so sorry skeletons, you're so misunderstood
+			if(bp)
+				bp.receive_damage(0, 0, 200)
+				playsound(M, get_sfx("desceration"), 50, TRUE, -1)
+				M.visible_message("<span class='warning'>[M]'s bones hurt too much!!</span>", "<span class='danger'>Your bones hurt too much!!</span>")
+				M.say("OOF!!", forced = "bonehurtingjuice")
+			else //SUCH A LUST FOR REVENGE!!!
+				to_chat(M, "<span class='warning'>A phantom limb hurts!</span>")
+				M.say("Why are we still here, just to suffer?", forced = "bonehurtingjuice")
+		else //you just want to socialize
+			if(bp)
+				playsound(M, get_sfx("desceration"), 50, TRUE, -1)
+				M.visible_message("<span class='warning'>[M] rattles loudly and flails around!!</span>", "<span class='danger'>Your bones so much your missing muscles spasm!!</span>")
+				M.say("OOF!!", forced="bonehurtingjuice")
+				bp.receive_damage(200, 0, 0) //But I don't think we should
+			else
+				to_chat(M, "<span class='warning'>Your missing arm aches from whever you left it.</span>")
+				M.say("*custom sighs", forced="bonehurtingjuice")
```

Comment:
M.emote("sigh")

Reply:
does emote have the forced parameter?  If it doesn't I'm going to keep it like that.

Reply:
I doubt anyone's going to check logs about a sigh, and we have tons of forced emotes already.